pkg/scheduler/framework/interface.go

```diff
@@ -740,6 +743,7 @@ func (p *PreFilterResult) Merge(in *PreFilterResult) *PreFilterResult {
 	}
 
 	r.NodeNames = p.NodeNames.Intersection(in.NodeNames)
+	r.UnschedulablePlugins = p.UnschedulablePlugins.Clone()
```

Comment:
how do we handle case when `in` has UnschedulablePlugins?

Reply:
`in` will be no UnschedulablePlugins